sample/lit-todo/index.html

```diff
@@ -1,76 +1,160 @@
-            ${result.map(row)}
-          </table>`,
-          document.body,
-        );
-      },
-    },
-  );
-
-  const toArray = async iterator => {
-    const arr = [];
-    for await (const entry of iterator) {
-      arr.push(entry);
-    }
-    return arr;
-  };
-</script>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Redo: Offline-First Todos with Replicache</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,500' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Material+Icons&display=block' rel='stylesheet'>
+    <link href='styles.css' rel='stylesheet'>
+
+    <script type='module'>
+        import Replicache from './node_modules/replicache/out/mod.js';
+        import { html, render } from 'lit-html';
+        import '@material/mwc-top-app-bar/mwc-top-app-bar.js';
+        import '@material/mwc-list/mwc-check-list-item.js';
+        import '@material/mwc-list/mwc-list.js';
+        import '@material/mwc-icon';
+        import '@material/mwc-fab';
+        import '@material/mwc-textfield';
+
+        var rep = new Replicache({
+            // URL of the diff server to use. The diff server periodically fetches
+            // the 'client view' from your service and returns any delta. You can
+            // use our hosted diff server (as here) or a local diff server, which
+            // is useful during development. See
+            // https://github.com/rocicorp/replicache#server-side for more
+            // information.
+            diffServerURL: 'https://serve.replicache.dev/pull',
+
+            // Auth token for the diff server, if any.
+            diffServerAuth: '1',
+
+            // URL of your service's Replicache batch endpoint. Replicache
+            // will send batches of mutations here for application when the
+            // network is available.
+            batchURL: 'https://replicache-sample-todo.now.sh/serve/replicache-batch',
+
+            // Auth token for your client view and batch endpoints, if any.
+            dataLayerAuth: '2',
+        });
+
+        rep.onSync = syncing => {
+            const icon = document.querySelector('.sync-icon');
+            icon.textContent = 'sync';
+            icon.classList.toggle('offline', false);
+            if (syncing) {
+                icon.classList.toggle('syncing', true);
+                window.setTimeout(() => icon.classList.toggle('syncing', false), 500);
+            }
+            if (!rep.online) {
+                icon.classList.toggle('offline', true);
+                icon.textContent = 'sync_problem';
+            }
+        };
+
+        const key = id => `/todo/${id}`;
+
+        const updateTodo = rep.register('updateTodo', async (tx, changes) => {
+            const todo = await tx.get(key(changes.id));
+            Object.assign(todo, changes);
+            await tx.put(key(todo.id), todo);
+        });
+
+        const deleteTodo = rep.register('deleteTodo', async (tx, { id }) => {
```

Comment:
please run `npm run format` or set vscode to format on save

Reply:
Done